README.md

```diff
@@ -30,7 +30,12 @@ There are two main categories of samples: **Camera** and **Applications**. The s
       - [**ReadIterateZDF**][ReadIterateZDF-url] - Read point cloud data from a ZDF file, iterate through it, and extract individual points.
       - [**ZDF2PLY**][ZDF2PLY-url] - Convert point cloud from ZDF a file to a PLY file.
   - **Advanced**
-    - [**HandEyeCalibration**][HandEyeCalibration-url] - Perform Hand-Eye calibration
+    - [**HandEyeCalibration**][HandEyeCalibrationReadme-url]
+        - [**HandEyeCalibration**][HandEyeCalibration-url] - Perform Hand-Eye calibration.
+        - [**UtilizeEyeInHandCalibration**][UtilizeEyeInHandCalibration-url] - Transform single data point or entire point cloud from camera frame to robot base frame using Eye-in-Hand calibration matrix.
+          - **Dependencies:**
```

Comment:
Did you consider using NuGet for this? Then:
- Everything would just work automatically for the user, they won't have to install anything manually
- They'll automatically get the correct version
- You don't need to document which dependencies are needed and where to get them (and this documentation can't accidentally get outdated)
- Packages get installed locally to the project, users don't need to install things globally on their system
- It will be possible to build in CI since we don't need to manually install stuff first
- This is how .NET users will expect things to work

Reply:
> Did you consider using NuGet for this?

That would be the ideal solution! How do I use NuGet this way? 
Is it enough to have package.config file in the sample folder? 

(Sorry for asking stupid questions, but i'm not familiar with C# and this is the first sample where we rely on some third-party packages)

Reply:
> If i have the package.config in the sample folder, they will automatically be downloaded by NuGet when I build the sample. This was very easy and required no manual work from me (as a user). Is this what you mean?

That is what I mean. If you add these dependencies using NuGet in Visual Studio, it will be set up to work like this automatically. When a user then clones this repo and builds using Visual Studio, they don't have to do anything manually.
 
> This is only possible in Windows I think, but I will need to check it out.

Our .NET api is Windows only, you should remove everything you write about Ubuntu from this PR.

Reply:
done